src/components/quiz-next.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Question 2</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/gh/Pratham82/min-ui@dev/src/css/minui.css"
+    />
+    <link rel="stylesheet" href="../css/index.css" />
+  </head>
+  <body>
+    <nav class="flex justify-around items-center navbar">
+      <div class="flex items-center branding">
+        <a href="../../../index.html">
```

Comment:
I think you can do '/index.html' instead of this '...//.../..'

Reply:
resolved